docs/serving/services/using-queue-extensions.md

```diff
@@ -0,0 +1,80 @@
+# Using extensions enabled by QPOptions
+
+Once your cluster is setup with extensions enabled by QPOptions, a Service can decide which extensions it wish to use and how to configure such extensions. Activating and configuring extensions is described here.
```

Comment:
It still doesn't explain what QPOptions is here - I would like to see a proper conceptual intro so that users know why they would enable this or what they would use it for. For example, something like:

"QPOptions is an optional Go package that provides extensions for Knative Services. These extensions allow you to..." etc

I don't think what's currently here is enough to warrant adding a new section to the docs.

Reply:
Added an intro to what is QPOptions